pom.xml

```diff
@@ -55,7 +55,7 @@
 
     <modules>
         <module>caf-logging-tomcat-juli</module>
-        <module>opensuse-tomcat-jul</module>
+        <module>opensuse-tomcat-juli</module>
```

Comment:
The suggestion was to rename the module opensuse-tomcat-juli-image to be consistent with what was here already. The existing module <module>caf-logging-tomcat-juli</module> produces a jar, the existing module <module>opensuse-tomcat-image</module> produces a docker image. As your new module produces a docker image then <module>opensuse-tomcat-juli-image</module> matches the existing convention.
Is there a reason to follow another here?

Reply:
@frankmccarry this was a mistake in my part, will correct

Reply:
@frankmccarry do you want the module name changed [here](https://github.com/CAFapi/opensuse-tomcat-image/blob/US590035a/opensuse-tomcat-juli/pom.xml#L29) also, or only the artifact being produced?

The name being camel case or lack of, if memory serves, was highlighted previously so I want to be sure of whether this change is required as well

Reply:
Hi @tbroadbent22, good catch. Were you thinking of changing 'openSUSE Tomcat juli' to be 'openSUSE Tomcat Juli image', capital J and add image?
Thanks, if you spot any more highlight them.

Reply:
Yes, I havent seen any others, so this should be the last of them